base/compiler/typeinfer.jl

```diff
@@ -173,6 +176,49 @@ function cache_result!(interp::AbstractInterpreter, result::InferenceResult, val
     nothing
 end
 
+function finish_opaque_closure!(clos::PartialOpaque, mod::Module, interp::AbstractInterpreter)
+    # Infer with the most general types possible, so that optimization has
+    # access to the result.
+
+    argt = unwrap_unionall(clos.t).parameters[1]
+
+    argtypes = Any[argt.parameters...]
+    pushfirst!(argtypes, clos.env)
+    if isdispatchtuple(argt)
+        # If we don't need to track specializations, just infer this here
+        # right now.
```

Comment:
```suggestion
        # If we don't need to track specializations, just infer this here right now.
```

Julia's line limit is 92 character so this fits on one line